specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -7819,6 +7819,11 @@
           "description": "The name of the vendor that raise the alert.",
           "readOnly": true,
           "type": "string"
+        },
+        "AlertLink": {
```

Comment:
you should also add this field to the example files (e.g GetSecurityAlertEntityById.json

Reply:
Done